glusterd2/xlator/options/options.go

```diff
@@ -186,12 +187,29 @@ func ValidateInt(o *Option, val string) error {
 
 // ValidateInternetAddress validates the Internet Address
 func ValidateInternetAddress(o *Option, val string) error {
-	return ErrInvalidArg
+	_, _, err := net.ParseCIDR(val)
+	if err != nil {
+		return ErrInvalidArg
+	} else if validate.IsIP(val) != true {
```

Comment:
Shouldn't validateInternetAddress also consider a valid hostname to be a correct val passed in?

Reply:
@atinmu validate.IsHost() takes care of that.

Reply:
My bad, sorry.